python/ray/actor.py

```diff
@@ -877,10 +877,9 @@ def _remote(self, args=None, kwargs=None, **actor_options):
         # If the actor methods require CPU resources, then set the required
         # placement resources. If actor_placement_resources is empty, then
         # the required placement resources will be the same as resources.
-        actor_placement_resources = {}
+        actor_placement_resources = resources.copy()
```

Comment:
Per our discussion, let's have a fix for all the language frontends.

Reply:
Actually I am still curious why we are using placement resources for the first scheduling. Is it intended?

Reply:
Resolved offline. Added a fix that works for all the frontends, and changed RequiredResources to PlacementResources in the current scheduling bug.